Compile error for feather m4

Hello. older version used to work but the newer one does not allow me to upload a sketch. here is what i get as an error:

```
Documents\Arduino\libraries\FlashStorage\src\FlashStorage.cpp: In function 'void invalidate_CMCC_cache()':

Documents\Arduino\libraries\FlashStorage\src\FlashStorage.cpp:56:13: error: 'struct Cmcc' has no member named 'CR'; did you mean 'SR'?

   if (CMCC->CR.bit.CSTS) {

             ^~

             SR
```
i modified CR.bit.CSTS to SR.bit.CSTS and it seems to work ok (example sketch works). is that allright or theres something else is going on. thanks

--- a/src/FlashStorage.cpp
+++ b/src/FlashStorage.cpp
@@ -53,7 +53,7 @@ static inline uint32_t read_unaligned_uint32(const void *data)
 // Invalidate all CMCC cache entries if CMCC cache is enabled.
 static void invalidate_CMCC_cache()
 {
-  if (CMCC->CR.bit.CSTS) {
+  if (CMCC->SR.bit.CSTS) {
     CMCC->CTRL.bit.CEN = 0;
     while (CMCC->SR.bit.CSTS) {}
     CMCC->MAINT0.bit.INVALL = 1;

